usaspending_api/etl/es_etl_helpers.py

```diff
@@ -587,10 +639,19 @@ def post_to_elasticsearch(client, job, config, chunksize=250000):
             printf({"msg": f"No documents to add/delete for chunk #{count}", "f": "ES Index", "job": job.name})
             continue
 
+        # Only delete before adding/inserting/indexing new docs on incremental loads, not full reindexes
+        is_incremental = config["is_incremental_load"] and str(config["is_incremental_load"]).lower() == "true"
```

Comment:
note to self: refactor this as it seems like Javascript. `config["is_incremental_load"]` is already set as a bool